content/en/building/reference/app-settings/_index.md

```diff
@@ -136,9 +136,27 @@ The following filter functions are available for formatting dates.
 
 
 ## Validations
+
 Validation rules are code fragments used to determine if some input is valid. For example, to say a field is only valid if the value has at least five characters, you would use the lenMin(5). They are used in `registrations[].validations.list[].rule` and `patient_reports[].validations.list[].rule` to determine if an incoming report is accepted. A report is accepted as valid only if all rules return `true`. If any validation rule returns `false` then the report is marked as invalid, and a message is automatically sent to the submitter. The content for the message is set in the `translation_key` associated to each rule. If a report fails multiple validations then each message will be sent. These can be combined into a single SMS by specifying `"*.validations.join_responses" : true`.
 
+The syntax aims to mimic C-like languages. You can use logical operators, ternaries, nested "blocks" (`rule && (some || nested || rules)`) and validation functions (`validationFunction("arg1", "arg2")`).
+
+**String parameters for validation functions, such as the regex in the "regex" function, should be quoted.**
+
+Non-quoted parameters will be cast to floats (numbers with decimals).
+
+For each validation function, there is also a matching function prepended by `other` that allows you to run functions on other values than the one the rule string is for. This can be useful for fields that have differing requirements depending on another field. For example:
+
+```json
+{
+  "state": "otherEquals('country', 'US') ? lenMin(2) : lenMin(0)"
+}
+```
```

Comment:
Can you add th explanation of this check as well.

Reply:
Added. I also realised we don't have any tests for "otherEquals" so I added some.